docs/recipes/useAtomEffect.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: useAtomEffect
```

Comment:
Do you want to add a link to `extensions/effect.mdx`?
Or, I was thinking of adding this recipe there.

Reply:
A link would be good.